### PR TITLE
Add projects to UserProfile

### LINKED ISSUE #43: Projects cards in UserProfile
add a subsection in profile for active voyages (smaller cards, niot the big ones)

### LINKED ISSUE #47: Fix title visibility in UserProfile editable text fields

### PR DESCRIPTION
**Pull request by thinktwice13**
### Summary
- Add project cards to UserProfile
screenshot from 2018-09-07 22-30-38
- Refactor
  - Get route param from `props.match.params` instead of App.js route wrapper
  - Turn into function
- Fix UserProfile sidebar titles visbility in edit mode
peek 2018-09-06 20-05

#### Todos
- Better styling
- Include projects not related to voyage teams
- Adjust voyage cards (make smaller and move?)

Closes #43 
Closes #47

### CODE REVIEW

File: src/components/UserProfile/index.jsx [hunk 1]
```diff
@@ -1,49 +1,79 @@
...
+              <img className="team-card-avatar-img" src={user.avatar ? user.avatar : require('../../assets/blank image.png')} alt={user.username} />
+              {/* <div className="team-card-username">{user.username}</div> */}
+            </Link>
+          )
+        })
+        break;
+      case 'TechStack':
+        data = info.data && info.data.map((tech, idx) => {
+          return (
+            <div key={idx} className="team-card-techstac k">{tech.name}</div>
```
Comment (the-vampiire) [inline src/components/UserProfile/index.jsx]: does this className need to be fixed?

File: src/components/UserProfile/index.jsx [hunk 2]
```diff
@@ -101,51 +129,205 @@ class UserProfile extends React.Component {
...
+export default props =>
   <Request
     {...props}
     query={profileQuery}
-    variables={props.username && { username: props.username }}
+    variables={{ username: props.match.params.username }}
     component={UserProfile}
     globalLoader
-  />)
+  />
```
Comment (the-vampiire) [inline src/components/UserProfile/index.jsx]: is this old code thats commented out?

File: src/components/UserProfile/index.jsx [hunk 3]
```diff
@@ -1,49 +1,79 @@
...
+// PROJECT CARD INFO //
+const InfoComponents = ({ team }) => {
+  const { project, cohort, tier, title } = team
+  const infoObjects = [
+    { label: 'Voyage Dates', data: dateFormatter(cohort.start_date) + " - " + dateFormatter(cohort.end_date) },
+    { label: 'Team Name', data: title },
+    { label: 'Project', data: project.title },
+    { label: 'Elevator Pitch', data: project.elevator_pitch },
+    { label: 'Tier', data: 'Tier ' + tier.level },
+    { label: 'Team', data: project.users },
```
Comment (the-vampiire) [inline src/components/UserProfile/index.jsx]: change to Members?

File: src/components/UserProfile/index.jsx [hunk 4]
```diff
@@ -1,49 +1,79 @@
...
-  }
+// PROJECT CARD INFO //
+const InfoComponents = ({ team }) => {
+  const { project, cohort, tier, title } = team
+  const infoObjects = [
+    { label: 'Voyage Dates', data: dateFormatter(cohort.start_date) + " - " + dateFormatter(cohort.end_date) },
+    { label: 'Team Name', data: title },
+    { label: 'Project', data: project.title },
+    { label: 'Elevator Pitch', data: project.elevator_pitch },
+    { label: 'Tier', data: 'Tier ' + tier.level },
```
Comment (the-vampiire) [inline src/components/UserProfile/index.jsx]: move Tier under Team Name

can Tier have the tier icon like in the mockup? @serpient

File: src/components/UserProfile/index.jsx (hunk 1 above)
Comment (thinktwice13) [reply src/components/UserProfile/index.jsx]: Yeah. Not used atm, so never saw the issue.

**Comment by thinktwice13** [comment]
@the-vampiire just took another look. Are you seeing a scrollbar *in* the card?

Another thing I just noticed is that the projects card extends outside of its `<section>` container because of its fixed width. Easy fix I think, but I'd like to see where we end up with your suggestion first.

**Comment by serpient** [comment]
should we rendering both voyage cards and project cards? can we just render project cards?
image

**Comment by the-vampiire** [comment]
we should move voyages to a list format rather than large cards for the profile. lets move that to a future idea 

save the cards for the /voyages view
in general i think they should shrink in size. i think more square shaped would work. there isnt much information on them that justifies the width imo

**Comment by the-vampiire** [comment]
@serpient what do you think about the text and moving the users to a bottom row of the Project card?

**Pull request merged**